### ISSUE TITLE
Translation for Spanish-speaking practitioners who are less comfortable with English

### ISSUE THREAD

**Issue by sjoerdly** [ScrumFacilitators/measuringoutcome-es#2]
As discussed in ScrumFacilitators/measuringoutcome-en#2 - First translation could cover only the instructions & other sentences, and leave KVA/KVM cards for later. 

Current state: no spanish translation available
Desired state: a variation of the PPT that has instructions and sentences in Spanish, except for KVA/KVM cards.

Acceptance criteria:

- [ ] Conveys the same meaning/intentions as the English material
- [x] Design is unchanged (except where necessary to potentially accommodate different text-lengths)
- [x] We put the spanish product in its own repo. So including translated readme.md
- [ ] The resulting work is reviewed by another spanish/english proficient speaker/reader.

Questions/to decide (potentially to be split off):

- Should the title of the game also be translated? I prefer not to as this is the 'product' identifier.
- Should the readme.md be translated right off the bat? 
- Should we include mention of the translation in the (english) readme.md?
- Should the license.md be translated as well, or is this unnecessary since translations of the CC licenses are available elsewhere?

**Comment by natastro** [comment]
Thank you so much for the update @cheehonghsia :)

**Comment by natastro** [comment]
> Hi @stroudn1 now I can tag you! The issue is assigned. I think we will be adopting your suggested approach of using a seperate repository per language.
> 
> Feel free to get started while we set that up. But again, no rush.
> 
> This does mean we'd want to translate the full repo (so also the readme.md) :-) We'll see about transferring the issues and setting up memberships there asap. :-)
> 
> Catch you on the flip side!
> 
> Side note: are you a member of our Scrum Facilitators Slack community yet? Couldn't find you there. If you're a Scrum Practitioner (which I assume as you want to help translate) feel free to connect on https://www.linkedin.com/in/sjoerdkranendonk/ & Let me know if you're not yet in the Slack and would like to join.

Hi @sjoerdly ! I connected with you on LinkedIn. I actually am not a Scrum Practitioner but have worked as a Project Administrator and know the PM I work with wants to learn about Scrum. I'm always excited to learn!

Thank you for more clarification again on what's being translated. I'll keep you guys in the loop!

**Comment by sjoerdly** [comment]
In creating the dutch translation issues, I recognize that this issue COULD be split further. Is that helpful @stroudn1 ?  Or is everything clear to you? 

Anyway, here is a screenshot of what we believe needs to be translated on the explanation slide. 
![image](https://user-images.githubusercontent.com/1508446/86468371-96424d80-bd37-11ea-812e-a2d3fd5cfef9.png)

**Comment by natastro** [comment]
Hi @sjoerdly ! This is clear, thank you so much! One quick question - this can be found in the 'en' (English) repo, correct?

**Comment by sjoerdly** [comment]
Yes! Which reminds me, we should also link from the readme to the english
repo... ;-)

AND DONE. used google translate so a human check is nice.

**Comment by sjoerdly** [comment]
All we need for the first release is translation of the last few cards. These are 'takeaways' so are not part of the 'external' EBM framework.

![Screenshot 2020-07-08 at 16 22 20](https://user-images.githubusercontent.com/1508446/86932284-747f0700-c139-11ea-9feb-18b4f3ceb9f1.png)

@stroudn1 is this something you are willing to do too? PS I embedded the fonts to the ppt so you can also see the design as intended.

**Comment by natastro** [comment]
@sjoerdly Absolutely! This is in the PowerPoint, correct?

**Comment by sjoerdly** [comment]
@stroudn1 yes, the last few slides.

**Comment by sjoerdly** [comment]
So basically all that needs to be done is for a second spanish speaker to review the translated materials and contrast it to the english one to catch potential interpretation differences. 

I'm breaking that off into a seperate task, as I have enough confidence in Natalies translation to make a first release. IF anything needs to be changed, we can make a new release :-)

**Comment by natastro** [comment]
Glad to hear this @sjoerdly ! Let me know if you need anything else translated!

**Comment by cheehonghsia** [comment]
Thanks for translating @natastro. I'll share this today on LinkedIn as well :)
